packages/vats/src/provisionPool.js

```diff
@@ -23,6 +23,8 @@ export const privateArgsShape = M.splitRecord(
   harden({
     // only necessary on first invocation, not subsequent
     initialPoserInvitation: InvitationShape,
+    // expected only in upgrade incarnations
+    metricsOverride: M.recordOf(M.string()),
```

Comment:
That's a little misleading... "upgrade" in a contract usually means restarting a vat with a new implementation associated with durable state.

This `metricsOverride` is about restarting _the whole swingset VM_ with all new vats.